### PR TITLE
Add Taskana app

### PR DESCRIPTION
**Pull request by iDuuck**

### CODE REVIEW

**Comment by shinnn** [comment]
You're missing some of the contribution guidelines. https://github.com/sindresorhus/awesome-electron/blob/d09b6b9c663355b9f4c68cfc0ba741387f04bfcd/contributing.md

File: readme.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Some good apps written with Electron.
 - [Kakapo](https://github.com/bluedaniel/kakapo-app) - An ambient sound mixer for relaxation or productivity.
 - [DevDogs](https://github.com/ragingwind/devdogs) - DevDocs.io API documentation.
 - [Blox Party](https://github.com/kvnneff/bloxparty) - Multiplayer Tetris game.
+- [Taskana](https://github.com/kreativgebiet/taskana) - Unofficial Asana App.
```
Comment (sindresorhus) [inline readme.md]: > Unofficial Asana client.

**Comment by arthurvr** [comment]
You'll also have to fix the merge conflict.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**